src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -162,9 +187,11 @@ def adam_transform(job, masterIP, inFile, snpFile, inputs):
                 "--conf", "spark.hadoop.fs.default.name=hdfs://%s:%s" % (masterIP, HDFS_MASTER_PORT),
                 "--conf", "spark.local.dir=/ephemeral/spark",
                 "--", "transform", 
-                "mkdups.adam", "ri.adam",
+                "hdfs://"+masterIP+":"+HDFS_MASTER_PORT+"/mkdups.adam", "hdfs://"+masterIP+":"+HDFS_MASTER_PORT+"/ri.adam",
```

Comment:
Can we split this line into two for readability?

Also, perhaps:

```
"hdfs://%s:%s/mkdups.adam" % (masterIP, HDFS_MASTER_PORT)
```

Instead of:

```
"hdfs://"+masterIP+":"+HDFS_MASTER_PORT+"/mkdups.adam"
```